### DISCUSSION TITLE
今天不知道为什么Mac的明日方舟一直出现网络配置失败问题

### DISCUSSION THREAD

**Discussion by wusongzhou** [MaaAssistantArknights/MaaAssistantArknights#5055] [Q&A] [answered]
<img width="1590" alt="image" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/98697034/577403b2-5ebb-4cfb-b34a-abf2bb92ec93">

**Comment by ABA2396** [comment]
这几天官服在推新的登录方式，估计是有验证签名啥的，应该只能用[AVD](https://developer.android.com/studio/run/managing-avds)了）

**Reply by wusongzhou** [reply]
那真是个悲伤的消息

**Reply by bagpipesuki** [reply]
请问 Mac 上用 AVD 的话，应该怎么设置呢？目前是无法启动自动战斗的😭
<img width="351" alt="截屏2023-06-06 22 47 35" src="https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/134058336/f809ed57-e604-4fef-a9fb-e9af68952d69">

**Comment by HongyuS** [comment, marked as answer]
找到解决办法了：
https://github.com/hguandl/PlayCover/issues/2#issuecomment-1579705417
（转载自 PlayCover 的 Discord 群）
